R/bm-write-csv.R

```diff
@@ -10,25 +10,39 @@ write_csv <- Benchmark(
   "write_csv",
   setup = function(source = names(known_sources),
                    writer = "arrow",
+                   compression = "uncompressed",
```

Comment:
Any reason not to put `c("uncompressed", "gzip")` here so all variations will get run by default?